src/App.tsx

```diff
@@ -177,8 +183,10 @@ const App = () => {
       if (!semver.valid(lastSeenReleaseNotesVersion) || semver.lt(lastSeenReleaseNotesVersion, version)) {
         setAppVersion(version);
         setShowReleaseNotes(true);
+        dispatch(handleWalkThroughStep(1));
```

Comment:
Can we use enum values instead of numbers?

Reply:
I'm also okay with strings instead of enums if we use a union type like `currentStep: "someStep" | "otherStep" | "coolStep"`

Reply:
@andreiv1992, agree! updated